README.md

```diff
@@ -101,10 +101,10 @@ arguments and options you expect. All other commandeer constructs (described bel
 are placed under it. They are all optional - although you probably want to use
 at least one, right?
 
-**subcommand `identifier`, `name`**
+**subcommand `identifier`, `name`** or **subcommand `identifier`,`names`**
 
 `subcommand` declares `identifier` to be a variable of type `bool` that is `true`
-if the first command line argument passed is `name` and is `false` otherwise.
+if the first command line argument passed is `name`(or is one of `names`) and is `false` otherwise.
```

Comment:
The according reformulation (also notice the added space):

if the first command line argument passed is `name` or one of the aliases (`alias1`, `alias2`, etc.) and is `false` otherwise.

Reply:
Let's also modify the example to make it clearer. It should read:

```nim
commandline:
  subcommand add, "add", "a":
    arguments filenames, string
    option force, bool, "force", "f"
  option globalOption, bool, "global", "g"

if add:
  echo "Adding", filenames
if globalOption:
  echo "Global option activated"
```